.travis.yml

```diff
@@ -31,6 +31,9 @@ cache:
   directories:
   - "$HOME/.npm"
 
+before_script:
+  - sed -i -e 's/SENTRY_RELEASE/$GIT_COMMIT_HASH/g' src/environments/environment.staging.ts
```

Comment:
So eventually isn't this `TRAVIS_COMMIT`? I guess adding it to before script doesn't make much sense, the script partis fine (this could be the first item). You should create a script, and dispatch on the branch. Replace environment.prod.ts on master and environment.staging.ts  in devel. No need to do replace in other branches.